SampleProject/VOKCoreDataManagerTests/ManagedObjectImportExportTests.m

```diff
@@ -170,15 +171,44 @@ - (void)testImportArrayWithCustomMapperOnWriteBlock
     [VOKCoreDataManager writeToTemporaryContext:^(NSManagedObjectContext *tempContext) {
         [VIPerson vok_addWithArray:array forManagedObjectContext:tempContext];
         dispatch_semaphore_signal(semaphore);
-    } completion:NULL];
+    } completion:nil];
     [self waitForResponse:1 semaphore:semaphore];
 
     NSArray *arrayOfPeople = [VIPerson vok_fetchAllForPredicate:nil forManagedObjectContext:nil];
     XCTAssertTrue([arrayOfPeople count] == 5, @"person array has incorrect number of people");
 
-    [arrayOfPeople enumerateObjectsUsingBlock:^(VIPerson *obj, NSUInteger idx, BOOL *stop) {
-        [self checkCustomMappingForPerson:obj andDictionary:[self makePersonDictForCustomMapper]];
-    }];
+    for (VIPerson *obj in arrayOfPeople) {
+        [self checkCustomMappingForPerson:obj
+                            andDictionary:[self makePersonDictForCustomMapper]];
+    }
+}
+
+- (void)testAsynchronousImportArrayWithCustomMapperReturningArrayOfManagedObjectIDs
+{
+    NSArray *array = @[[self makePersonDictForCustomMapper],
+                       [self makePersonDictForCustomMapper],
+                       [self makePersonDictForCustomMapper],
+                       [self makePersonDictForCustomMapper],
+                       [self makePersonDictForCustomMapper]];
+    VOKManagedObjectMapper *mapper = [VOKManagedObjectMapper mapperWithUniqueKey:nil andMaps:[self customMapsArray]];
+    [[VOKCoreDataManager sharedInstance] setObjectMapper:mapper forClass:[VIPerson class]];
+
+    dispatch_semaphore_t semaphore = dispatch_semaphore_create(0);
```

Comment:
are you using this instead of XCTestExpectation because it's what is used in the rest of the tests?